src/redux/books/booksSlice.js

```diff
@@ -1,40 +1,73 @@
-import { createSlice } from '@reduxjs/toolkit';
-
-const initialState = [
-  {
-    itemId: 'item1',
-    title: 'The Great Gatsby',
-    author: 'John Smith',
-    category: 'Fiction',
-  },
-  {
-    itemId: 'item2',
-    title: 'Anna Karenina',
-    author: 'Leo Tolstoy',
-    category: 'Fiction',
-  },
-  {
-    itemId: 'item3',
-    title: 'The Selfish Gene',
-    author: 'Richard Dawkins',
-    category: 'Nonfiction',
-  },
-];
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const API_BASE_URL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi';
+const KEY = '7u3HEQWYdcjxX54hw874';
+const FETCH_BOOKS = 'books/fetchBooks';
+const ADD_BOOKS = 'books/addBook';
+const REMOVE_BOOK = 'books/removeBook';
+
+export const fetchBooks = createAsyncThunk(FETCH_BOOKS, async () => {
```

Comment:
- [ ] You did a great job refactoring the add and remove functions. :+1: After reloading the page all the books added beforehand are gone. Please make sure the books appear when the page loads as suggested in the project requirements. ✍️

**Project requirements**
![image](https://github.com/shm04/bookstore/assets/17243325/ca0cefe1-fdfe-4394-ab00-14ed70b9c90b)